core.yaml

```diff
@@ -376,7 +376,7 @@ adminApps:
       - svc: thanos-receive
         port: 19291
         namespace: monitoring
-        paths: [/api/v1/receive/]
+        paths: [/api]
```

Comment:
According to Thanos docs the path should be `/api/v1/receive`. Could you tell us the reason of exposing whole thanos-receive API  ?
See: https://thanos.io/v0.28/components/receive.md/#example

Reply:
Setting path to /api/v1/receive in ingress and vs results in 404

Reply:
ok, then it sounds indeed that more is needed. Thanks